packages/headless/src/api/service/case-assist/case-assist-params.test.ts

```diff
@@ -65,7 +65,9 @@ describe('case assist params', () => {
 
   describe('prepareSuggestionRequestFields', () => {
     it('should structure fields in API format', () => {
-      const effective = prepareSuggestionRequestFields({myField: 'some value'});
+      const effective = prepareSuggestionRequestFields({
```

Comment:
When the API client was written, it was intentional to set the value directly on the property. The purpose was to make it easier to use by removing the unnecessary complexity of the `value` level.

Please check if `CaseAssistApiClient` was exposed to end users before your changes. If yes, then the change would be considered breaking.

Reply:
It may be slightly easier to use in the unlikely scenario that someone is using the client without any controllers right now.

But when using the client with values from the state, which is 99% of the case, it forces us to parse the state into this previous format to pass as arguments, then the arguments get parsed again immediately into exactly the same shape as before in order to be sent to the API.

Considering the state is holding this data and the API request parameters have the same shape. I think this is more suitable.

I don't see the CaseAssistAPIClient being explicitly exposed but if you could be my second opinion on this it would be greatly appreciated. Thanks!